lib/Core/BuildEngine.cpp

```diff
@@ -965,6 +965,8 @@ class BuildEngineImpl {
     
     // Find the cycle by searching from the entry node.
     struct WorkItem {
+      WorkItem(Rule * node) { this->node = node; }
+
```

Comment:
Nope, that doesn't work. This is because of a C++ 14 feature (Aggregate initialization with default member initializers) that isn't supported until VS 2017.